### PR TITLE
update ce_vrrp to fix a bug

### PR DESCRIPTION
**Pull request by yanzhangi**
##### SUMMARY

Manages VRRP interface attributes on HUAWEI CloudEngine devices
When a parameter is of the bool type, for example: If the value is True or yes, an error is reported in the module.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

lib/ansible/modules/network/cloudengine/ce_vrrp.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/network/cloudengine/ce_vrrp.py [hunk 1]
```diff
@@ -48,8 +48,7 @@
     admin_ignore_if_down:
         description:
             - mVRRP ignores an interface Down event.
-        type: bool
-        default: 'no'
```
Comment (yuandongx) [inline lib/ansible/modules/network/cloudengine/ce_vrrp.py:52]: miss type defined.
The blow is same as this.

**Review by yuandongx — CHANGES_REQUESTED** [review]
Missed type

**Comment by xuxiaowei0512** [comment]
shipit

**Comment by xuxiaowei0512** [comment]
bot_status

**Comment by yanzhangi** [comment]
read_for_review

**Comment by hwDCN** [comment]
shipit

**Pull request merged**